### DISCUSSION TITLE
Capitalisation and math mode in title

### DISCUSSION THREAD

**Discussion by alexander-betts** [retorquere/zotero-better-bibtex#2710] [Q&A] [answered]
I want to cite a paper with the title "Quadratic Chabauty and rational points I: _p_-adic heights" where the "p" is in math mode. How should I store this entry in Zotero, so that the capitalisation and math mode are both preserved when I export to (Better) BibTeX?

Putting $ signs around the p and giving the entry the #LaTeX tag gets the math mode right, but not the capitalisation. Is there a better way to do this?

**Comment by retorquere** [comment, marked as answer]
If you tag it as `#LaTeX`, I'm just passing the input through, which is why you see no title-casing.

Zotero doesn't have math support, so there's just three ways I know:

* `Quadratic Chabauty and rational points I: 𝑝-adic heights` (where `𝑝` is the [mathematical italic small P](https://www.compart.com/en/unicode/U+1D45D); I will have to make a change to BBT to have that export as `$p$`), or
* `Quadratic Chabauty and rational points I: <span class="nocase"><i>p</i></span>-adic heights` for an approximation, or
* `Quadratic Chabauty and rational points I: <script>$p$</script>-adic heights` to mark a part of the title as LaTeX

The first two will work in Zotero too, but that 3rd will mess with Zotero-generated bibliographies.

**Reply by alexander-betts** [reply]
Thanks! These are all great suggestions -- I think that, given the propensity of mathematicians in my field to use characters like [\mathbb{Q}](https://www.compart.com/en/unicode/U+211A) in titles, the third option would work best for what I'm looking for, even though the other Zotero functionality may suffer. But I'd certainly be pleased for the functionality in the first point, if that gets added to BBT at some point.

**Reply by retorquere** [reply]
I'm going to add the change, should not be too hard.

**Reply by retorquere** [reply]
Turns out it already works.
